GUI/GUI.csproj

```diff
@@ -50,47 +50,47 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\mzLib.1.0.325\lib\net471\Chemistry.dll</HintPath>
+    <Reference Include="Chemistry, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
```

Comment:
Are you building with x64? I believe MSIL is for "Any CPU" so that MM can work on x32 computers...

Reply:
![image](https://user-images.githubusercontent.com/16883585/42648730-5998321c-85cd-11e8-87b9-47570430a536.png)
There's no option to make the other to "Any CPU"

Reply:
I couldn't get it to work, so I manually edited GUI.csproj in notepad :DumpsterFire:

Reply:
Thanks for working on that, Zach.